bin/test_pylint

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash -e
+#
+# Run pylint on all the detected python files and compare against the expected list.
+# This is far from perfect, but the goal is just to prevent horrible things and to get
+# people to at least think about the style of python code. Adding a new python file will
+# require people to add another line in the test_pylint.out golden file.
```

Comment:
There are only a handful of lint errors - it's probably worth marking them as ignored  in the file or resolving them to save the additional step of adding the file to the